static/app/views/app/index.tsx

```diff
@@ -270,25 +272,40 @@ function App({children, params}: Props) {
       <LastKnownRouteContextProvider>
         <RouteAnalyticsContextProvider>
           {renderOrganizationContextProvider(
-            <AsyncSDKIntegrationContextProvider>
-              <GlobalFeedbackForm>
-                <GlobalDrawer>
-                  <MainContainer tabIndex={-1} ref={mainContainerRef}>
-                    <GlobalModal onClose={handleModalClose} />
-                    <SystemAlerts className="messages-container" />
-                    <Indicators className="indicators-container" />
-                    <ErrorBoundary>{renderBody()}</ErrorBoundary>
-                  </MainContainer>
-                </GlobalDrawer>
-              </GlobalFeedbackForm>
-            </AsyncSDKIntegrationContextProvider>
+            <ChonkThemeProvider>
+              <AsyncSDKIntegrationContextProvider>
+                <GlobalFeedbackForm>
+                  <GlobalDrawer>
+                    <MainContainer tabIndex={-1} ref={mainContainerRef}>
+                      <GlobalModal onClose={handleModalClose} />
+                      <SystemAlerts className="messages-container" />
+                      <Indicators className="indicators-container" />
+                      <ErrorBoundary>{renderBody()}</ErrorBoundary>
+                    </MainContainer>
+                  </GlobalDrawer>
+                </GlobalFeedbackForm>
+              </AsyncSDKIntegrationContextProvider>
+            </ChonkThemeProvider>
           )}
         </RouteAnalyticsContextProvider>
       </LastKnownRouteContextProvider>
     </Profiler>
   );
 }
 
+function ChonkThemeProvider({children}: {children: React.ReactNode}) {
+  const organization = useOrganization({allowNull: true});
```

Comment:
There’s one render cycle where `organization` is still `null` when this renders. This makes sense because the `OrganizationProvider` doesn’t itself load the organization.

We could alternatively move the `ChonkThemeProvider` further down the tree - e.g. into the `OrganizationLayout` below the `OrganizationContainer`. There, accessing the flag would be safe. But I think since we have a boolean flag, working with a falsy value is also fine.